mlrun/datastore/redis.py

```diff
@@ -163,3 +163,7 @@ def rm(self, key, recursive=False, maxdepth=None):
                 self.redis.delete(k)
         else:
             self.redis.delete(key)
+
+    @property
+    def spark_url(self):
+        return ""
```

Comment:
Doesn't look right - should be same as url()

Reply:
`store_path_to_spark()` does this:
```python
schemas = ["redis://", "rediss://", "ds://"]
if any(path.startswith(schema) for schema in schemas):
    url = urlparse(path)
    if url.path:
        path = url.path
```
If you simplify that, it's just `""` (since the path is appended to the result by the caller).